pandas/tests/computation/test_pytables.py

```diff
@@ -0,0 +1,23 @@
+from typing import Any
+
```

Comment:
add a test here: pandas/tests/io/pytables/test_categorical.py

pls follow the samples there. this needs to be a user facing test, use the example from the OP

Reply:
done